src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/CalledLegacySegmentCollection.java

```diff
@@ -0,0 +1,107 @@
+package org.broadinstitute.hellbender.tools.copynumber.formats.collections;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.formats.metadata.SampleLocatableMetadata;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.CalledCopyRatioSegment;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.CalledLegacySegment;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Represents a CBS-style segmentation to enable IGV-compatible plotting.
+ *
+ * IGV ignores column headers and requires that no other headers are present.
+ * We use the conventional CBS-style column headers (which includes the sample name)
+ * and suppress the SAM-style metadata header (which breaks the contract for construction from input files).
+ * See <a href="http://software.broadinstitute.org/cancer/software/genepattern/file-formats-guide#CBS">
+ *     http://software.broadinstitute.org/cancer/software/genepattern/file-formats-guide#CBS</a>
+ * and <a href="https://software.broadinstitute.org/software/igv/SEG">
+ *     https://software.broadinstitute.org/software/igv/SEG</a>.
+ *
+ */
+public final class CalledLegacySegmentCollection extends AbstractSampleLocatableCollection<CalledLegacySegment> {
+    //note to developers: repeat the column headers in Javadoc so that they are viewable when linked
+    /**
+     * Sample, Chromosome, Start, End, Num_Probes, Call, Segment_Mean
```

Comment:
Is this the canonical order?  I thought `Call` followed `Segment_Mean` in previous pipelines, since it was just appended to the CBS segment file.

Reply:
If you want IGV to be able to read it properly, the segment mean must be the last column.  I think the legacy format was sub-optimal and people using IGV tended to grab the seg file before the call file was appended.

No action.